### ISSUE TITLE
Symbols

### ISSUE THREAD

**Issue by churstz** [DuendeArchive/IdentityModel#300]
Based on this (https://github.com/IdentityModel/IdentityModel.OidcClient/issues/26) my assumption is Symbols files are/were being uploaded to http://srv.symbolsource.org/pdb/Public This server appears to be flaky at best.  As the server isn't up at the time of posting I have no way to confirm the symbols actually made it.

Is this assumption correct on where the symbols are uploaded? 

I would like to request that a better more stable option might be used such that the symbols reside alongside the nuget packages when published

https://devblogs.microsoft.com/nuget/improved-package-debugging-experience-with-the-nuget-org-symbol-server/

**Comment by leastprivilege** [comment]
I think the symbols are part of the Nuget  package.

**Comment by churstz** [comment]
My apologies you are correct.  I ended up going the long way around on this.  

As an FYI, I was getting a VerificationException coming out of the RequestUrlExtensions.CreateAuthorizeUrl() call.  I was able to trace this down to the QueryHelpers.AddQueryString() methods use of UrlEncoder.Default.Encode().

Internally the System.Text.Encoding.Web.TextEncoder.Encode is making use of non type safe code (i.e. C#'s unsafe) which requires permissions to run.  The quick solution is to add FullTrust but less than desirable.
